src/libsystemd-network/ndisc-router.c

```diff
@@ -275,6 +276,14 @@ int sd_ndisc_router_get_hop_limit(sd_ndisc_router *rt, uint8_t *ret) {
         return 0;
 }
 
+int sd_ndisc_router_get_retrans_time(sd_ndisc_router *rt, usec_t *ret) {
```

Comment:
Please use `uint64_t` instead of `uint64_t`, as you can see `sd_ndisc_router_get_icmp6_ratelimit()` below.

Reply:
I understand the intent of your comment is to replace usec_t with uint64_t. Done.

Reply:
Ah, yeah.